zips/zip-0271.md

```diff
@@ -141,7 +154,10 @@ to
 ## Extension to the lockbox funding stream
 
 The ``FS_DEFERRED`` lockbox funding stream is set to receive
-12% of the block subsidy and is extended until block height 4406400.
+12% of the block subsidy and is extended by 1,260,000 blocks on
+both Mainnet and Testnet. If this ZIP activates before or on the first block after the
+end of current dev fund as of this writing and specified in [^zip-1015-funding-streams],
+this would extend the ``FS_DEFERRED`` on Mainnet until block height 4406400.
```

Comment:
```suggestion
this would extend the ``FS_DEFERRED`` stream on Mainnet until block height 4406400.
```
or
```suggestion
this would extend ``FS_DEFERRED`` on Mainnet until block height 4406400.
```